man/systemd.service.xml

```diff
@@ -898,6 +898,18 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RestartQuick=</varname></term>
```

Comment:
hmm, i sense we might need to extend this later, hence maybe call this RestartMode= or so, and make it an enum, which takes "normal" or "quick" as values?

Reply:
(background: i am not entirely sure going through "deactivating" state is really always desirable, i.e. running ExecStop= and friends. hence we might need to extend this later)